src/installation/guides/fde.md

```diff
@@ -136,34 +135,28 @@ Copy the RSA keys from the installation medium to the target root directory:
 # cp /var/db/xbps/keys/* /mnt/var/db/xbps/keys/
 ```
 
-Before we enter the chroot to finish up configuration, we do the actual install.
-Do not forget to use the [appropriate repository
-URL](../../xbps/repositories/index.md#the-main-repository) for the type of
-system you wish to install.
+Before entering the chroot to finish up configuration, it is time to perform the actual install.
+Do not forget to use the [appropriate repository URL](../../xbps/repositories/index.md#the-main-repository) 
+for the type of system you wish to install.
+
+Legacy BIOS systems use the `grub` package instead of `grub-x86_64-efi`. For LVM users, The `lvm2` package is required.
 
 ```
-# xbps-install -Sy -R https://repo-default.voidlinux.org/current -r /mnt base-system lvm2 cryptsetup grub
-[*] Updating `https://repo-default.voidlinux.org/current/x86_64-repodata' ...
-x86_64-repodata: 1661KB [avg rate: 2257KB/s]
-130 packages will be downloaded:
-...
+# xbps-install -Sy -R https://repo-default.voidlinux.org/current -r /mnt base-system cryptsetup grub-x86_64-efi
 ```
 
-UEFI systems will have a slightly different package selection. The installation
-command for a UEFI system will be as follows.
+Before entering the chroot, Copy a DNS configuration file from live system to allow internet connection in the chroot.
 
 ```
-# xbps-install -Sy -R https://repo-default.voidlinux.org/current -r /mnt base-system cryptsetup grub-x86_64-efi lvm2
+# cp /etc/resolv.conf /mnt/etc/
```

Comment:
Let's clean this up a bit:

For UEFI systems, also install necessary GRUB support.

```
xbps-install -Sy -R https://repo-default.voidlinux.org/current -r /mnt grub-x86_64-efi
```